src/pages/dashboard/components/menu/menu.component.tsx

```diff
@@ -1,7 +1,50 @@
 import React from 'react'
 
-import { MenuStyled } from './menu.style.tsx'
+import { BsHandbag } from 'react-icons/bs'
+import { CiSettings } from 'react-icons/ci'
+import { FiUsers, FiFolderMinus } from 'react-icons/fi'
+import { IoMdLogOut } from 'react-icons/io'
+import { IoChatbubbleEllipsesOutline } from 'react-icons/io5'
+import { TbCategory } from 'react-icons/tb'
+
+import {
+  MenuStyled,
+  Logo,
+  MenuItem,
+  MenuList,
+  LogoutButton
+} from './menu.style.tsx'
 
 export const Menu: React.FC = () => {
-  return <MenuStyled>menu</MenuStyled>
+  return (
+    <MenuStyled>
+      <Logo>
+        <img src="Metrix.png" alt="logo"></img>
+      </Logo>
+      <MenuList>
```

Comment:
according to task:
![image](https://github.com/rrasta/dashboard/assets/7860124/fcc83780-3784-4fa2-ab8a-6457eb0eaf16)

menu should have just:
- dashboard
- Users
- Settings